app/routes/users/components/Penalties.js

```diff
@@ -22,11 +25,17 @@ function Penalties({ penalties }: Props) {
       {penalties.map(penalty => {
         const word = penalty.weight > 1 ? 'prikker' : 'prikk';
         return (
-          <li key={penalty.id}>
+          <li key={penalty.id} style={{ paddingBottom: '10px' }}>
             <strong>
-              {penalty.weight} {word}:{' '}
+              Du har fått {penalty.weight} {word}
             </strong>
-            {penalty.reason}
+            <br />
+            Begrunnelse: <i>{penalty.reason}</i>
+            <br />
+            Utgår:{' '}
+            <i>
+              <FormatTime time={moment(penalty.exactExpiration)} />
```

Comment:
Looks like the `<FormatTime />` component converts the `time` prop to a moment instance automatically. https://github.com/webkom/lego-webapp/blob/387b70e6bb357326d05dfa6798d5a11fe6791dbe/app/components/Time/index.js#L52

So you can write `<FormatTime time={penalty.exactExpiration} />` instead.

Reply:
Nice! I'll get onto that.